Software/Velvet_ESP32/lib/ESP32_STM_UART/include/ESP32_STM_UART.h

```diff
@@ -0,0 +1,138 @@
+/**
+ * @file ESP32_STM_UART.h
+ * @brief Заголовочный файл класса ESP32_STM_UART для обработки коммуникации с STM32.
+*/
+
+#ifndef ESP32_STM_UART_H
+#define ESP32_STM_UART_H
+
+#include <Arduino.h>
+#include <HardwareSerial.h>
+
+/**
+ * @class ESP32_STM_UART
+ * @brief Класс для обработки коммуникации с STM32 по протоколу, основанному на UART.
+ * Данный класс реализует протокол обмена данными между микроконтроллерами ESP32 и STM32 с помощью интерфейса UART.
+ * Он предоставляет ряд методов для отправки и получения команд и данных, в том числе:
+ * * проверка наличия подключения к Wi-Fi;
+ * * обработка входящих сообщений;
+ * * отправка данных на сервер;
+ * * проверка версии прошивки;
+ * * отправка ответов на запросы передачи весовых данных;
+ * * отправка метки времени в формате unix timestamp;
+ * * передача данных прошивки на STM.
+*/
+
+class ESP32_STM_UART {
+  public:
+/**
+ * @brief Конструктор класса
+ * @param serial Интерфейс UART, который будет использоваться для коммуникации.
+ * @param baudRate Скорость передачи данных в бодах.
+*/
+  ESP32_STM_UART(HardwareSerial &serial, uint32_t baudRate);
```

Comment:
What's wrong with your indentation? 
Use Ctrl+Alf+F in VSCode before committing your changes.

Reply:
Can't understand what you mean. Please don't forget that this is my first c/cpp project of this level. Don't be angry